doc/contributing.md

```diff
@@ -1,12 +0,0 @@
-# Contributing to Ruby
-
-This guide outlines ways to get started with contributing to Ruby:
-
-* [Reporting issues](contributing/reporting_issues.md): How to report issues, how to request features, and how backporting works
```

Comment:
I think the new documentation removes the link on how to report issues.

Reply:
Fixed.